source/fundamentals/crud/write-operations/update.txt

```diff
@@ -0,0 +1,293 @@
+.. _csharp-update-guide:
+
+================
+Update Documents
+================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you'll learn how to edit existing documents or upsert new documents 
+in your MongoDB collections using update operations.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``restaurants`` collection
+from the ``sample_restaurants`` database. The documents in this
+collection use the following ``Restaurant`` class as a model:
+
+.. literalinclude:: /includes/fundamentals/code-examples/crud/update/Restaurant.cs
+   :language: csharp
+   :dedent:
+   :start-after: start-model
+   :end-before: end-model
+
+This collection is from the :atlas:`sample datasets </sample-data>` provided
+by Atlas. See the :ref:`<csharp-quickstart>` to learn how to create a free MongoDB cluster
+and load this sample data.
+
+Update Operations
+-----------------
+
+You can perform update operations in MongoDB with the following methods:
+
+- ``UpdateOne()``, which updates *the first document* that matches the query filter.
+- ``UpdateMany()``, which updates *all documents* that match the query filter.
+
+Required Parameters
+~~~~~~~~~~~~~~~~~~~
+
+Update operations require two parameters: 
+
+- A **query filter** document, which determines which records to update. See the 
+  :manual:`MongoDB server manual </core/document/#query-filter-documents>` for 
+  more information about query filters.
+- An **update** document, which specifies the **update operator** (the kind of update to
+  perform) and the fields and values that should change. See the 
+  :manual:`MongoDB server manual</reference/operator/update-field/>` for a complete 
+  list of update operators and their usage. 
+
+The {+driver-short+} includes a ``Builders`` class that simplifies the creation of both
+query filters and update documents. The following code sample uses ``Builders`` to create
+a query filter that searches for restaurants with a ``borough`` field value of "Manhattan". 
+Then it uses ``Builders`` to create an update document that sets the value of the 
+``borough`` field of these restaurants to "Manhattan (north)":
```

Comment:
```suggestion
query filters and update documents. The following code sample uses ``Builders`` to create:
- A filter that matches documents with the ``borough`` field value "Manhattan"
- An update document that sets the value of the ``borough`` field to "Manhattan (north)"
```

Reply:
Changed to a list, but reworded your suggestion to avoid introducing the items with a sentence fragment.